Header header
string partName
string version
string serialString
float64 voltage1
float64 voltage2
float64 voltage3
float64 current
float64 coulombs
float64 temp



